### PR TITLE
Fix problem with lvk

### PR DESCRIPTION
**Pull request by adrianbayer**
There was a problem when plotting the powerspectra. lvk on disagreed with lvk off.

FIXED by changing the power.py python script to weight the particles by their masses (as needed for the lvk case because each ncdm has a different mass).

Note: Some other changes to the ncdm code were made along the way.

### CODE REVIEW

**Comment by adrianbayer** [comment]
I did some weird stuff with tracking ncdm.lua which you can ignore.

I think it's all good to merge, but will wait for you to confirm.

**Comment by rainwoodman** [comment]
There is a conflict -- is it realistic that you do a rebase first?

File: libfastpm/thermalvelocity.c [hunk 1]
```diff
@@ -237,19 +235,19 @@ fastpm_ncdm_init_create(
     FastPMncdmInitData* nid = malloc(sizeof(nid[0]));
 
     nid->BoxSize = BoxSize;
-    nid->Omega_ncdm = 0.001404; /* FIXME: use new cosmology.c */
+    nid->Omega_ncdm = 0.001404; /* FIX use new cosmology.c */
```
Comment (rainwoodman) [inline libfastpm/thermalvelocity.c]: vim will automatically highlight FIXME -- FIXME and NOTE are somewhat standardlized comment keywords.

**Comment by rainwoodman** [comment]
Also, could you add a summary in the PR's comment (click on edit) to describe the problem it fixes and why the CL fixes it? 

Without the context we will not recall what this actually do -- actually I already forgot what this is about..

**Comment by rainwoodman** [comment]
```
git checkout -b ncdm-backup ncdm # a back up in case the rebase is messed up
git checkout ncdm
git fetch upstream
git rebase upstream/master
```
It will likely then tell you there are errors. You can look for '>>>>' marks and fix them.

rebase will tell you what to do when you are done with the files.

After `git rebase --continue` is done, you can do a force push to update this PR.
```
git push -f
```

**Comment by adrianbayer** [comment]
Okay all done, thanks!

**Comment by rainwoodman** [comment]
Yay!

**Pull request merged**